multi-area-model.ipynb

```diff
@@ -57,7 +57,7 @@
     "* [S0. Configuration](#section_0)\n",
     "* [S1. Parameterization](#section_1)\n",
     "    * [1.1. Parameters to tune](#section_1_1)\n",
-    "    * [1.2. Default parameters](#section_1_2)\n",
+    "    * [1.2. Configuring main and additional parameters](#section_1_2)\n",
```

Comment:
I would suggest moving this module to S2 so that tuning parameters for users would stand out. And this 1.2 could be changed to 2.1 and titled Configuring model parameters.

Reply:
I agree, it is a good idea. Could you make this change?

Reply:
Yes, let's make this happen for the next PR when cleaning the code and only focus on the parameter setting and documentation.